pages/docs/manual/latest/primitive-types.mdx

```diff
@@ -166,6 +166,95 @@ var result = 1 + 2;
 
 </CodeTab>
 
+## Big Integers (experimental)
+
+**Since 11.1**
+
+For values which are too large to be represented by Int or Float, there is the `bigint` primitive type.
+We provide the usual operations on them: `+`, `-`, `*`, `/`, etc. See [BigInt](api/core/bigint) for helper functions.
+
+A `bigint` number is denoted by a trailing `n` like so: `42n`.
+
+As `bigint` is a different data type than `int`, it's necessary to open the corresponding module to overload the operators.
+
+<CodeTab labels={["ReScript", "JS Output"]}>
+
+```res example
+open! Js.BigInt
```

Comment:
I'd prefer to just use `open! Bigint` when RescriptCore is opened, but that does not work yet, a new Core version is needed.